src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <Display result="12" />
+      <ButtonPanel />
+    </div>
```

Comment:
Opps, we need to use `fragment` here

We don't want to create an extra element, that is why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.

![image](https://user-images.githubusercontent.com/31897434/104458762-129dcf00-55ac-11eb-95de-a8010c94e594.png)

https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376016/